.travis.yml

```diff
@@ -7,7 +7,19 @@ compiler:  gcc
 
 before_install:
 - sudo apt-get update -qq
-- sudo apt-get install -qq libperl4-corelibs-perl libtool xorg-dev doxygen python-dev alien swig libgtest-dev libstdc++6 libxml2-dev gettext libxinerama-dev libxft-dev libxrandr-dev libxcursor-dev libgdbm-dev libc6 libc6-dev libmng-dev zlib1g-dev libcap-dev libpng12-0 libpng12-dev freeglut3-dev flex libx11-dev bison++ bisonc++ libjpeg-dev libjpeg8-dev python2.7 python2.7-dev python2.7-psycopg2 libogdi3.2-dev libgif-dev libxerces-c-dev libgeos-dev libgeos++-dev libfreetype6 libfreetype6-dev python-imaging libproj-dev python-setuptools libgif-dev libxerces-c-dev libcap-dev libpq-dev openssl libxml2-utils libxmu-dev python-git cmake
+- sudo apt-get install -qq
+    alien bison++ bisonc++ cmake doxygen freeglut3-dev flex gettext
```

Comment:
For the curious, I verified that the only meaningful changes in these lines are the addition of `libboost-dev` and `libyaml-cpp-dev`.

Reply:
The others are needed. I just wanted to point out what the changes were so others don't have to compare the lists again.

Reply:
ok @tst-lsavoie